public/locales/en/reservation.json

```diff
@@ -1,7 +1,20 @@
 {
+  "hour": "{{count}} hour",
+  "hour_other": "{{count}} hours",
+  "minute": "{{count}} minutes",
```

Comment:
```suggestion
  "minute": "{{count}} minute",
```
This should probably also use the singular form?

Reply:
Fixed